src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -70,6 +72,8 @@ import './ActivitySidebar.scss';
 type ExternalProps = {
     activeFeedEntryId?: string,
     activeFeedEntryType?: FocusableFeedItemType,
+    activityFilterAllTitle?: ActivityFilterAllTitle,
```

Comment:
Why do we need to pass this down?

Reply:
This is related to "All Comments" vs "All Activities" title. There are alternative options to passing it down such as determining the title based on characteristics in ActivitySidebarFilter or ActivitySidebar.